console/pom.xml

```diff
@@ -175,6 +175,10 @@
       <artifactId>commons-validator</artifactId>
     </dependency>
     <!-- Spring dependencies -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-autoconfigure</artifactId>
+    </dependency>
```

Comment:
I'm concerned by mixing spring boot dependencies on a classic spring project

Reply:
> I'm concerned by mixing spring boot dependencies on a classic spring project

@groldan is it safe to do so ?

Reply:
confirmed by @groldan , we can't mix spring boot dependencies into regular spring applications, so we will have to rework this.

Reply:
OK, but why it is not possible ?